eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -59,6 +63,23 @@ func (c *ClusterLoader) Apply() (err error) {
 			Ready:     true,
 		},
 	}
+
+	//Wait for job to complete -- 2 hours because larger tests take a very long time.
+	timeout := 2 * time.Hour
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	var job *v1.Job
+	for ok := true; ok; ok = job.Status.Succeeded < 1 {
```

Comment:
```
job := v1.Job{}
for job.Status.Succeeded < 1 {
}
```

Reply:
This code block produces an error, as the following Get() command in the code needs a *v1.Job for assignment, cannot do direct assignment.
Tried doing the following, but that also results in a seg-fault. I think this is the way to keep it, although looks a little bit confusing.
`var job *v1.Job
*job = v1.Job{}`

Reply:
How about
```
job := &v1.Job{}
for job.Status.Succeeded < 1 {
}
```
This will make job a job pointer, fixing your type issue

Reply:
Ah yes, thanks!